doc/release-process.md

```diff
@@ -61,62 +63,97 @@ The release script has the following dependencies:
 You can optionally install the `progressbar2` Python module with pip to have a
 progress bar displayed during the build process.
 
-## Release process
+## Versioning
 
-In the commands below, <RELEASE> and <RELEASE_PREV> are prefixed with a v, ie.
-v1.1.0 (not 1.1.0).
+Zcashd uses semantic version numbers as described by https://semver.org/.
```

Comment:
It does not, and never has. In particular, we release breaking software changes via what SemVer would call "minor revisions", which is incompatible with SemVer. Remove this section, or change it to explain what we actually do.